.travis.yml

```diff
@@ -1,18 +1,15 @@
 sudo: required
 dist: trusty
 services:
-  - docker
+- docker
 language: go
 go:
-  - 1.8.3
+- 1.8.3
 script:
-  - make docker-test
+- make docker-test
 notifications:
-  email:
-    recipients:
-      - eng@portworx.com
-      - aditya@portworx.com
-    on_success: always
     on_failure: always
```

Comment:
This is good.  You may want to consider adding
*on_success: change*
That means, you will get notified:
1.  For every failure
2.  When a build succeeds after the previous one failed